lib/src/widgets/chat_widget.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+import 'package:flutter_chat_ui/src/models/message.dart';
+
+import 'message_view.dart';
+import 'profile_circle.dart';
+
+class ChatWidget extends StatelessWidget {
+  const ChatWidget({
+    Key? key,
+    required this.message,
+    required this.isCurrentUser,
+    required this.onLongPress,
+    this.senderProfileImage,
+    this.outgoingChatBubbleColor,
+    this.inComingChatBubbleColor,
+    this.chatBubbleTextStyle,
+    this.linkStyle,
+    this.onShareButtonPressed,
+    this.onImageViewTap,
+    this.shareIcon,
+    this.shareIconBackgroundColor,
+    this.heightOfImageMessage,
+    this.widthOfImageMessage,
+    this.inComingChatBubbleBorderRadius,
+    this.outgoingChatBubbleBorderRadius,
+  }) : super(key: key);
+
+  final Message message;
+  final bool isCurrentUser;
+  final String? senderProfileImage;
+  final Color? outgoingChatBubbleColor;
+  final Color? inComingChatBubbleColor;
+  final TextStyle? chatBubbleTextStyle;
+  final TextStyle? linkStyle;
+  final void Function()? onShareButtonPressed;
+  final void Function()? onImageViewTap;
+  final void Function(double) onLongPress;
+  final Icon? shareIcon;
+  final Color? shareIconBackgroundColor;
+  final double? heightOfImageMessage;
+  final double? widthOfImageMessage;
+  final BorderRadiusGeometry? inComingChatBubbleBorderRadius;
+  final BorderRadiusGeometry? outgoingChatBubbleBorderRadius;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
```

Comment:
if padding and margin are required then should use sized box.

Reply:
In flutter 2.5 sizedbox does not provide margin and padding